### PR TITLE
Add context flag to save removed devices

### PR DESCRIPTION
**Pull request by fdanis-oss**
Type of pull request:

- [ ] New plugin (Please include [new plugin checklist](https://github.com/fwupd/fwupd/wiki/New-plugin-checklist))
- [x ] Code fix
- [ ] Feature
- [ ] Documentation

This commit is linked to hughsie/libgusb#89

### CODE REVIEW

File: libfwupdplugin/fu-context.h [hunk 1]
```diff
@@ -54,6 +54,15 @@ typedef void (*FuContextLookupIter)(FuContext *self,
  **/
 #define FU_CONTEXT_FLAG_SAVE_EVENTS (1u << 0)
 
+/**
+ * FU_CONTEXT_FLAG_SAVE_REMOVED_DEVICES:
+ *
+ * Save removed devices so that they can be replayed to emulate devices.
+ *
+ * Since: 1.8.10
```
Comment (hughsie) [inline libfwupdplugin/fu-context.h:62]: can you make this 1.8.11 please, i'll tag a release today of 1.8.11 and we can merge after that is tagged.

Comment (fdanis-oss) [reply libfwupdplugin/fu-context.h]: Yes, done

**Comment by fdanis-oss** [comment]
@hughsie is there anything preventing to merge this?

**Comment by hughsie** [comment]
@fdanis-oss I don't think we need it; at least where I see the emulation work heading we either never want the removed devices, or we want both flags.

**Pull request closed without merging**